frontend/src/api/prometheus/distributedWorkloads.ts

```diff
@@ -49,12 +49,12 @@ export type WorkloadCurrentUsage = {
 };
 
 export type WorkloadWithUsage = {
-  workload: WorkloadKind | 'other';
+  workload: WorkloadKind;
   usage: number | undefined;
 };
 export type TopWorkloadsByUsage = Record<
   keyof WorkloadCurrentUsage,
-  { totalUsage: number; topWorkloads: WorkloadWithUsage[] }
+  { totalUsage: number; topWorkloads: WorkloadWithUsage[]; otherUsage: number | undefined }
```

Comment:
I am wondering if we should be looking at making this a type 🤔 I assume this is passed around... it's a type that is at least updated twice in this file alone